examples/trellis_barley.json

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Classic Trellis Example.",
+  "description": "Classic Barleys Example. Shows barley yields for diffent years across varieties and sites.",
```

Comment:
Why "classic barley example”?  This is a classic trellis plot in the paper by Becker and Cleveland.  No one knows it by the name “Barley” except ones who are familiar with Vega examples.